plugins/Observability/src/Observability.cpp

```diff
@@ -1845,3 +1832,32 @@ void Observability::showReport(bool b)
 	}
 }
 
+bool Observability::isMoon(QList<StelObjectP> & objectSelection)
+{
+   return !objectSelection.empty() && "Moon" == objectSelection[0]->getEnglishName();
+}
+
+bool Observability::isSun(QList<StelObjectP> & objectSelection)
+{
+   return !objectSelection.empty() && "Sun" == objectSelection[0]->getEnglishName();
+}
+
+bool Observability::isInterstellar(QList<StelObjectP> & objectSelection)
```

Comment:
This should be called isStar() or isFixed(). 

We have "Interstellar objects" now, i.e. moving bodies, and they are special types of planets. But this method actually separates planets from other types.

Reply:
Changed back to `isStar`.